studio/src/lib/schema-helpers.ts

```diff
@@ -50,6 +55,8 @@ export type ParsedGraphQLField = {
   name: string;
   description?: string;
   deprecationReason?: string;
+  authenticated?: boolean;
```

Comment:
Is there a reason this can't always be defined? Default false or whatever?

Reply:
I see that we typically do this with the other properties in general, but then they're propagated as default values (property || '') sort of thing. That was happening before this PR of course, but I do wonder what the benefits are?

Reply:
The reason was when I started prototyping there were a few places that created this structure but as I made a single point to just forgot to/didn't think of changing it to non-optional